test/runnable/mangle_prefix.d

```diff
@@ -0,0 +1,92 @@
+/+
+REQUIRED_ARGS: -mangle-prefix=.:_kek_ -mangle-prefix=runnable/mangle_prefix.d:_mypackage_._v2_0_
```

Comment:
I think there should also be a test for the main use case: prefix a package with a symbol, not only a single module.